### DISCUSSION TITLE
Subscription Not Found for Co-Administrator

### DISCUSSION THREAD

**Discussion by kiwidyne** [rcl-ssl/docs#34] [General]
I have a secondary account that has every permission imaginable to my MPN subscription and to the RCL app. But when I log into the RCL app, I receive an error message: "Subscription Not Found". I'm about at the end of my rope here. I'm trying to deal with an expired certificate for a client's app - unfortunately, I'm also trying to recover from surgery, so I have limited time to spend fooling around with the dumpster fire that is Microsoft account management.

**Comment by kiwidyne** [comment]
The previously mentioned account is actually a co-administrator for this MPN account.

**Reply by rcladmin** [reply]
Try this-
1. Create a new user account or use an existing account in azure active directory that has access to the subscription that contains the web app.
2. Assign owner of contributor role of the account to the azure subscription containing the web app.
3. Add this user account to the work or school account is the subscription section
4. Sign in with this work or school account

Follow the instruction on the page and its related links:

https://docs.rclapp.com/authorization/sign-in-accounts.html

**Reply by rcladmin** [reply]
Another possible cause for the error, is that your a signing in with an account that is a co-admin to the Azure account containing the web app. This account may be coming from another tenant, say you are managing an Azure account for a client. 

You could try using an account in the AAD tenant of the Azure account that contains the web app as a work/school account to login. Add the necessary access control and enter this account in the work/school account section of the subscription in the RCL portal.

The account that you login must be from the Azure account that contains the web app. Hope, this helps to solve this issue.
